packages/ses/src/error/tame-v8-error-constructor.js

```diff
@@ -260,7 +262,7 @@ export const tameV8ErrorConstructor = (
       if (errorTaming === 'unsafe') {
         const stackString = stackStringFromSST(error, sst);
         weakmapSet(stackInfos, error, { stackString });
-        return `${error}${stackString}`;
+        return `${errorToString(error)}${stackString}`;
```

Comment:
Not really more safe because it still defers to the `name` and `message` properties, but more aligned to what I understand is the behavior of the `.stack` property on Node and v8 which ignore an overridden `toString()`.

Reply:
Nice. Resolving.